### DISCUSSION TITLE
Exporting the raw Markdown file from a page over HTTP

### DISCUSSION THREAD

**Discussion by rgov** [gollum/gollum#1904] [Q&A] [answered]
If I want to grab the raw, unrendered Markdown from a page, is there an API for that? Like the "Raw" button on GitHub:

https://raw.githubusercontent.com/gollum/gollum/master/CONTRIBUTING.md

I understand I can access the underlying file on disk, or use the Edit button and copy the contents of the page.

**Comment by bartkamphorst** [comment, marked as answer]
There's no API for that as far as I know. I do think it could be a nice addition, so feel free to open a feature request or a pull request for this!
